maintainers/maintainer-list.nix

```diff
@@ -16258,6 +16258,12 @@
     githubId = 863327;
     name = "Tyler Benster";
   };
+  tbidne = {
+    email = "tbidne@protonmail.com";
+    github = "tbidne";
+    githubId = 2856188;
+    name = "Thomas Bidne";
+  };
```

Comment:
@tbidne it looks like you are not part of the @NixOS/nixpkgs-maintainers team, you likely missed the invite when you became maintainer / the team was created. This means you can't be requested for review properly. Please consider requesting another invite, see https://github.com/NixOS/nixpkgs/tree/master/maintainers#tools-for-maintainers for how to do that.